src/tlo/methods/healthsystem.py

```diff
@@ -1767,6 +1794,44 @@ def call_and_record_never_ran_hsi_event(self, hsi_event, priority=None):
                  priority=priority,
                  )
 
+    def write_to_hsi_no_blank_footprint_log(
+        self,
+        event_details: HSIEventDetails,
+        person_id: int,
+        facility_id: Optional[int],
+        did_run: bool,
+        priority: int,
+    ):
+        """Write the log `HSI_Event` and add to the summary counter only if the appointment footprint is not blank."""
```

Comment:
The docstring of function `write_to_hsi_no_blank_footprint_log()` needs to be updated with a description of each of the parameters. I have given a rough example below:
```suggestion
        """Write the log `HSI_Event` and add to the summary counter only if the appointment footprint is not blank.
        :param event_details: What event_details is responsible for
        :param person_id: The person_id of interest
        :param facility_id: What this is responsible for
        :param did_run: What this is responsible for
        :param priority: The priority of for the HSI event. Highest is..., Lowest is...."""
```